Improve Pairlist docs (#3331)
freqtrade/freqtrade: docs/configuration.md

When not using orderbook (`ask_strategy.use_order_book=False`), the price at the `ask_strategy.price_side` side (defaults to `"ask"`) from the ticker will be used as the sell price.

## Pairlists and Pairlist Handlers

Pairlist Handlers define the list of pairs (pairlist) that the bot should trade. They are configured in the `pairlists` section of the configuration settings.

In your configuration, you can use Static Pairlist (defined by the [`StaticPairList`](#static-pair-list) Pairlist Handler) and Dynamic Pairlist (defined by the [`VolumePairList`](#volume-pair-list) Pairlist Handler).

> Additionaly, [`PrecisionFilter`](#precision-filter), [`PriceFilter`](#price-pair-filter) and [`SpreadFilter`](#spread-pair-filter) act as Pairlist Filters, removing low-value pairs.

Reviewer: ```suggestion
Additionaly, [`PrecisionFilter`](#precision-filter), [`PriceFilter`](#price-pair-filter) and [`SpreadFilter`](#spread-pair-filter) act as Pairlist Filters, removing certain pairs.
```

I'd keep this open - as in theory, PriceFilter could implement a HighPrice filter (no pair > 100 USDT) - and noone would touch THIS section of the docs then (only the section about PriceFilter).

> If multiple Pairlist Handlers are used, they are chained and a combination of all Pairlist Handlers forms the resulting pairlist the bot uses for trading / backtesting / downloading historical data. Pairlist Handlers are executed in the sequence they are configured. You should always configure either `StaticPairList` or `VolumePairList` as the starting Pairlist Handler.

Reviewer: ```suggestion
If multiple Pairlist Handlers are used, they are chained and a combination of all Pairlist Handlers forms the resulting pairlist the bot uses for trading / backtesting. Pairlist Handlers are executed in the sequence they are configured. You should always configure either `StaticPairList` or `VolumePairList` as the starting Pairlist Handler.
```

data download does not use pairlists ...

> Inactive markets and blacklisted pairs are always removed from the resulting pairlist. Blacklisted pairs (those in the `pair_blacklist` configuration setting) are also always removed from the resulting pairlist.

Reviewer: This line has almost the same sentence twice. 
maybe as follows?

```suggestion
Inactive markets are always removed from the resulting pairlist. Explicitly Blacklisted pairs (those in the `pair_blacklist` configuration setting) are also always removed from the resulting pairlist.
```

#### Volume Pair List

`VolumePairList` employs sorting/filtering of pairs by their trading volume. I selects `number_assets` top pairs with sorting based on the `sort_key` (which can only be `quoteVolume`).

> !!! Deprecation

Author: I'm not sure I properly declared the deprecation warning block here, need to search if we have other similar notes in the docs...

Reviewer: This will not work.

we got the link to the documentation in the developer settings (or you can use [this one](https://squidfunk.github.io/mkdocs-material/extensions/admonition/#types) )- which lists available types (deprecation is not one of these).

honestly, i'd just keep this section out - i've intentionally not documented this here as i find "deprecated" documentation confusing for new users. 

what's that mean? can i use it - or better not? ... see what i mean?
